blog/2021-09-01-tdex-terraform-automation.md

```diff
@@ -56,6 +58,67 @@ $ ./deploy.sh
 ```
 ![Deploy](../static/img/deploy.png)
 
+### TOR 
 
-Since it is good practice to have backup enabled, please provide S3 bucket name when deploying. 
+Executing deploy.sh, you can setup your service behind TOR too. 
+You can provide your public Onion key, or if left empty, service will generate it for you. 
+If generated by TOR service, we advice you to check your TOR onion public key and register it.
+
+Connect to your Linux box over SSH and get onion key.
+```sh
+$ ssh -i ~/.ssh/key_location ubuntu@your.tdex.host.ip
+$ docker exec tor onions
+
+```
+
+Once you have your Onion public key, please follow [this](https://dev.tdex.network/docs/provider/registry) tutorial on how to register your Onion endpoint with us. 
+
+
+
+### Use TDEX CLI to interact with tdexd endpoint
+
+TDEX is a CLI (GRPC client), that can be used to interact with remote TDEX-deamon endpoint. 
+We need to init/configure tdex-cli, which is way of authenticating to deamon. 
+When configuring service, we need to provide several things:
+ - tls_cert_path
+ - macaroons_path
+ - rpcserver (this will be IP of machine where tdex-box is deployed)
 
+To install Tdex cli you need to clone tdex-deamon repository and build TDEX cli. 
```

Comment:
Lets provide a bash installer script to pull down the binary already compiled for his own machine